deepchem/hyper/__init__.py

```diff
@@ -111,3 +111,6 @@ def hyperparam_search(self,
     log("train_score: %f" % train_score, self.verbose)
     log("validation_score: %f" % best_validation_score, self.verbose)
     return best_model, best_hyperparams, all_scores
+
+
+from deepchem.hyper.gaussian_process import GaussianProcessHyperparamOpt
```

Comment:
I would like to but basically `GaussianProcessHyperparamOpt` is a subclass of the `HyperparamOpt` defined in this file, moving the import line to the top will cause error here. Should we just move the `HyperparamOpt` here to another file, like `grid_search.py`?